chore: Add description for region parameter (#31)
wherobots/airflow-providers-wherobots: README.md

The arguments for the `WherobotsRunOperator` constructor:

* `region: Region`: The Wherobots Compute Region where the runs will be executed on.
  The values available can be found in `wherobots.db.region.Region`.

>   The default value is `Region.AWS_US_WEST_2` and this is the only region Wherobots Cloud operate workloads on today.

Reviewer: ```suggestion
  The default value is `Region.AWS_US_WEST_2` and this is the only region in which Wherobots Cloud operates workloads today.
```

  > [!IMPORTANT]

>   > We'll start to enforce the region parameter as soon as Wherobots Cloud extend to support more regions and clouds.

Reviewer: ```suggestion
  > We will soon enforce the `region` parameter when Wherobots Cloud extends support to new regions and cloud providers.
```

The arguments for the `WherobotsRunOperator` constructor:

* `region: Region`: The Wherobots Compute Region where the runs will be executed on.
  The values available can be found in `wherobots.db.region.Region`.
  The default value is `Region.AWS_US_WEST_2` and this is the only region in which Wherobots Cloud operates workloads today.
  > [!IMPORTANT]
  > We will soon enforce the `region` parameter when Wherobots Cloud extends support to new regions and cloud providers.

>   > It's recommended to set the value now to avoid breaking your production tasks.

Author: updated

The arguments for the `WherobotsRunOperator` constructor:

* `region: Region`: The Wherobots region where runs are hosted.
  The values available can be found in `wherobots.db.region.Region`.
  The default value is `Region.AWS_US_WEST_2` and this is the only region in which Wherobots Cloud operates workloads today.
  > [!IMPORTANT]
  > To prepare for the expansion of Wherobots Cloud to new regions and cloud providers, the `region` parameter will become mandatory in a future SDK version.
  > Before this support for new regions is added, we will release an updated version of the SDK. 
  > If you continue using an older SDK version, your existing Airflow tasks will still work.

>   > However, any new jobs you create without specifying the `region` parameter will be hosted in the `aws-us-west-2` region.

Reviewer 2: ```suggestion
  > However, any new or existing jobs that don't specify the `region` parameter will be hosted in the `aws-us-west-2` region..
```

The arguments for the `WherobotsRunOperator` constructor:

* `region: Region`: The Wherobots region where runs are hosted.
  The values available can be found in `wherobots.db.region.Region`.
  The default value is `Region.AWS_US_WEST_2` and this is the only region in which Wherobots Cloud operates workloads today.

>   > [!IMPORTANT]
>   > We will soon enforce the `region` parameter when Wherobots Cloud extends support to new regions and cloud providers.
>   > We will version the SDK in advance of supporting new regions.
>   > In the new version of the SDK a region will be required rather than an optional parameter.
>   > If you remain on the current SDK version, your Airflow Task will not be impacted.
>   > But your Job run will always be hosted on the aws-us-west-2 region
>   > if you leave the `region` parameter unspecified.

Reviewer 2: ```suggestion
To prepare for the expansion of Wherobots Cloud to new regions and cloud providers, the `region` parameter will become mandatory in a future SDK version.
Before this support for new regions is added, we will release an updated version of the SDK. 
If you continue using an older SDK version, your existing Airflow tasks will still work. However, any new jobs you create without specifying the `region` parameter will be hosted in the `aws-us-west-2` region.
```

> * `region: Region`: The Wherobots region where runs are hosted.
>   The values available can be found in `wherobots.db.region.Region`.
>   The default value is `Region.AWS_US_WEST_2` and this is the only region in which Wherobots Cloud operates workloads today.
>   > [!IMPORTANT]
>   > We will soon enforce the `region` parameter when Wherobots Cloud extends support to new regions and cloud providers.
>   > We will version the SDK in advance of supporting new regions.
>   > In the new version of the SDK a region will be required rather than an optional parameter.
>   > If you remain on the current SDK version, your Airflow Task will not be impacted.
>   > But your Job run will always be hosted on the aws-us-west-2 region
>   > if you leave the `region` parameter unspecified.

Reviewer 2: ```suggestion
* `region: Region`: The Wherobots region where runs are hosted.
  The values available can be found in `wherobots.db.region.Region`.
  The default value is `Region.AWS_US_WEST_2` and this is the only region in which Wherobots Cloud operates workloads today.
  > [!IMPORTANT]
  > To prepare for the expansion of Wherobots Cloud to new regions and cloud providers, the `region` parameter will become mandatory in a future SDK version.
  > Before this support for new regions is added, we will release an updated version of the SDK. 
  > If you continue using an older SDK version, your existing Airflow tasks will still work.
  > However, any new jobs you create without specifying the `region` parameter will be hosted in the `aws-us-west-2` region.
```

  * `sparkExecutorDiskGB: int`: The disk size for the Spark executor.
  * `sparkConfigs: dict`: A dictionary of Spark configurations.
  * `dependencies: list[dict]`: A list of dependant libraries to install.

> For more detailed information about the `environment` parameter,

Reviewer: ```suggestion

For more detailed information about the `environment` parameter,
```

  * `sparkExecutorDiskGB: int`: The disk size for the Spark executor.
  * `sparkConfigs: dict`: A dictionary of Spark configurations.
  * `dependencies: list[dict]`: A list of dependant libraries to install.
  
  For more detailed information about the `environment` parameter,

>   please refer to the [Wherobots Documentation](https://docs.wherobots.com/latest/references/runs/?h=run#get-run-logs).

Reviewer 2: ```suggestion
  please refer to the [Wherobots Documentation](https://docs.wherobots.com/latest/references/runs/#get-run-logs).
```

The existing link will highlight every instance of the word "run" on the page.

> [!IMPORTANT]

> > Wherobots Community Edition users only have access to the "Tiny" runtime type.

Reviewer 2: Please provide the link to upgrade: https://docs.wherobots.com/latest/get-started/upgrade-organization/

> > Wherobots Community Edition users only have access to the "Tiny" runtime type.

Author: @dwyliebot @kadolor Thanks for the suggestions, updated, please help take a look again

  * `sparkExecutorDiskGB: int`: The disk size for the Spark executor.
  * `sparkConfigs: dict`: A dictionary of Spark configurations.
  * `dependencies: list[dict]`: A list of dependant libraries to install.
  
  For more detailed information about the `environment` parameter,

>   please refer to the [Wherobots Documentation](https://docs.wherobots.com/latest/references/runs/#get-run-logs).

Reviewer 2: ```suggestion
refer to  [Get Run Logs](https://docs.wherobots.com/latest/references/runs/#get-run-logs) in the Wherobots Documentation.
```

> [!IMPORTANT]

> > Today Wherobots Cloud offers COMMUNITY users free access to [tiny](https://docs.wherobots.com/latest/develop/runtimes/#runtime-specifications-chart) runtime.
> > If you need access to runtimes larger than `tiny`,
> > try Wherobots Cloud's pay-as-you-go [PROFESSIONAL](https://wherobots.com/pricing/) plan.
> > For more information, please refer to the [Guidance for upgrading your account](https://docs.wherobots.com/latest/get-started/upgrade-organization/).

Reviewer 2: ```suggestion
> Today, Wherobots Cloud offers free access to the ["Tiny" runtime](https://docs.wherobots.com/latest/develop/runtimes/#runtime-specifications-chart) through the Community Edition Organization.
> If you need access to larger runtimes (including Memory-Optimized and GPU-Optimized runtimes), consider upgrading to a Professional Edition Organization, Wherobots Cloud's [pay-as-you-go](https://wherobots.com/pricing/) plan.
> For more information, refer to the [Upgrade Organization](https://docs.wherobots.com/latest/get-started/upgrade-organization/) guidance in the Wherobots Documentation.
```

### Arguments

* `region: Region`: The Wherobots Compute Region where the SQL query will be executed on.
  The values available can be found in `wherobots.db.region.Region`.
  The default value is `Region.AWS_US_WEST_2` and this is the only region Wherobots Cloud operate workloads on today.
  > [!IMPORTANT]
  > We'll start to enforce the region parameter as soon as Wherobots Cloud extend to support more regions and clouds.

>   > It's recommended to set the value now to avoid breaking your production tasks.

Reviewer: Same comments/suggestions as above.

### Arguments

> * `region: Region`: The Wherobots Compute Region where the SQL query will be executed on.

Reviewer 2: @dwyliebot I think that this is for `WherobotsSqlOperator` and the prior set of arguments was for `WherobotsRunOperator`.

### Arguments

* `region: Region`: The Wherobots Compute Region where the SQL query executions are hosted.
  The values available can be found in `wherobots.db.region.Region`.
  The default value is `Region.AWS_US_WEST_2` and this is the only region in which Wherobots Cloud operates workloads today.

>   > [!IMPORTANT]
>   > We will soon enforce the `region` parameter when Wherobots Cloud extends support to new regions and cloud providers.
>   > We will version the SDK in advance of supporting new regions.
>   > In the new version of the SDK a region will be required rather than an optional parameter.
>   > If you remain on the current SDK version, your Airflow Task will not be impacted.
>   > But your SQL query will always be hosted on the aws-us-west-2 region
>   > if you leave the `region` parameter unspecified.

Reviewer 2: ```suggestion
  > [!IMPORTANT]
  > To prepare for the expansion of Wherobots Cloud to new regions and cloud providers, the `region` parameter will become mandatory in a future SDK version.
  > Before this support for new regions is added, we will release an updated version of the SDK. 
  > If you continue using an older SDK version, your existing Airflow tasks will still work.
  > However, any new or existing SQL queries that don't specify the `region` parameter will be hosted in the `aws-us-west-2` region.
```
